schema/transportation/segment.yaml

```diff
@@ -19,12 +19,24 @@ properties:
     unevaluatedProperties: false
     required: [subtype]
     allOf:
+      - title: "Segment Properties"
+        required: [class]
+        properties:
+          class: { "$ref": "#/$defs/propertyDefinitions/roadClass" }
+          access_restrictions: { "$ref": "#/$defs/propertyContainers/accessContainer" }
```

Comment:
I have a very soft preference for using `access` instead of `access_restrictions` for no good reason other than that it's shorter. But I can live with both.

Reply:
No strong preference for me either, I'll take a poll on this in the tf-transportation meeting tomorrow

Reply:
Resolving this for now because it's not pressing and we can change it later. I think there's more value in merging the change first.